tests/acceptance/ContactManagementCest.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+class ContactManagementCest
+{
+    public function _before(AcceptanceTester $I)
+    {
+        $I->login('admin', 'Maut1cR0cks!');
+    }
+
+    public function createContactFromQuickAdd(AcceptanceTester $I)
+    {
+        // Navigate to the contacts page
+        $I->amOnPage('/s/contacts');
+
+        // Wait for and click on the "Quick Add" button
+        $I->waitForElementClickable('#toolbar .quickadd', 30);
+        $I->click('#toolbar .quickadd');
+
+        // Wait for the Quick Add modal to appear and verify its presence
+        $I->waitForElementVisible('#MauticSharedModal-label', 30);
+        $I->see('Quick Add', 'h4.modal-title');
+
+        // Wait for the form fields to be visible
+        $I->waitForElementVisible('#lead_firstname', 10);
+
+        // Fill out the quick add form
+        $I->fillField('#lead_firstname', 'QuickAddFirstName');
+        $I->fillField('#lead_lastname', 'QuickAddLastName');
+        $I->fillField('#lead_email', 'quickadd@example.com');
+        $I->fillField('#lead_tags_chosen input', 'TestTag');
+        $I->pressKey('#lead_tags_chosen input', Facebook\WebDriver\WebDriverKeys::ENTER);
+        $I->fillField('#lead_companies_chosen input', 'Mautic');
+        $I->pressKey('#lead_companies_chosen input', Facebook\WebDriver\WebDriverKeys::ENTER);
+
+        // Wait for the save button to be clickable and submit the form
+        $I->waitForElementClickable('#MauticSharedModal > div > div > div.modal-footer > div > button.btn.btn-default.btn-save.btn-copy', 30);
```

Comment:
This seem fragile. If a div will be added or removed from the modal then this will break. Can it be reduced to
```suggestion
        $I->waitForElementClickable('#MauticSharedModal .btn-save', 30);
```
It's also improving readability of the code.

Reply:
Yes, I need to simplify the CSS selectors. I'll take care of it in future tests.